core/themes/basis/css/component/hero.css

```
@@ -26,6 +26,15 @@
   margin-top: -2rem;/* Negative top margin collapses spacing under header */
```

Maybe fix this code comment while we're here.

```suggestion
  margin-top: -2rem; /* Negative top margin collapses spacing under header */
```